### PR TITLE
Install latest version of Istio by default

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by ldemailly** [comment]
use 0.6.0 it's much better

**Comment by geoand** [comment]
@sdake @ldemailly Is there a way to "link" a PR in this repo with a PR in the docs repo? Because in cases such as these were one makes a change in the code that also requires a doc change, one needs to manually "track" the relation between the two PRs

**Comment by ldemailly** [comment]
usually you put it in the description of both, benefit of those being editable

in general though if you can't have it all on either repo (either just doc; or just generated code that ends up in doc automatically) the dependencies are best kept one direction and we kept backward compatible

**Comment by geoand** [comment]
Thank you @ldemailly !

**Comment by hklai** [comment]
@sdake I suppose it's because this branch does not have the OWNERS file in the install dir.
See https://github.com/geoand/istio/tree/62b8f396854eb06ceab8ea1830e5ee1132c4f138/install

It was added 4 days ago.
https://github.com/istio/istio/commit/eb9dcf35cfdc3c2b1d69d829d1e2c1a9483f9a23#diff-afdd44c6c511e6433c26993f95560d5a

A rebase is likely going to address that...

File: install/ansible/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The full list of configurable parameters is as follows:
 | `cluster_flavour` | Defines whether the target cluster is a Kubernetes or an Openshift cluster. | Valid values are `k8s` and `ocp` (default) |
 | `github_api_token` | The API token used for authentication when calling the GitHub API | Any valid GitHub API token or empty (default) |
 | `cmd_path` | Can be used when the user does not have the `oc` or `kubectl` binary on the PATH | Defaults to expecting the binary is on the path | 
-| `istio.release_tag_name` | Should be a valid Istio release version. If left empty, the latest Istio release will be installed | `0.2.12`, `0.3.0`, `0.4.0` (default) |
+| `istio.release_tag_name` | Should be a valid Istio release version. If left empty, the latest Istio release will be installed | `0.2.12`, `0.3.0`, `0.4.0` |
```
Comment (ldemailly) [inline install/ansible/README.md]: should this have ,... so we don't keep updating it ?
`0.2.12`, ..., `0.6.0`, ...

File: install/ansible/istio/defaults/main.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ github_api_token: ""
 
 istio:
 
-  # Could be a tag "0.2.12, 0.3.0, 0.4.0" version or be empty "", then in this case, the latest release will be downloaded
-  release_tag_name: "0.4.0"
+  # Could be a tag "0.2.12, 0.3.0, 0.4.0, 0.5.0, 0.5.1" version or be empty "", then in this case, the latest release will be downloaded
```
Comment (ldemailly) [inline install/ansible/istio/defaults/main.yml]: ditto

**Comment by ldemailly** [comment]
latest is 0.6.0 since a few weeks, and 0.7.0 soon enough

**Comment by geoand** [comment]
I updated the docs to indicate that the versions are open ended in order to avoid having to update them for every release

**Comment by ldemailly** [comment]
merging unrelated text files

**Pull request merged**